crm_stage_multi_company/views/crm_stage.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2019 ACSONE SA/NV
+     License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="crm_stage_form">
+        <field name="name">crm.stage.form (in crm_stage_multi_company)</field>
+        <field name="model">crm.stage</field>
+        <field name="inherit_id" ref="crm.crm_stage_form" />
+        <field name="arch" type="xml">
+            <field name="team_id" position="before">
+                <field
+                    name="company_id"
+                    options="{'no_open': True, 'no_create': True}"
+                    groups="base.group_multi_company"
+                />
+                <field name="company_id" invisible="1" />
+            </field>
+        </field>
+    </record>
+
+
```

Comment:
could you add the field also 
- on tree view , with option="hide"
- on search view 

thanks !